Tests/MatrixTests/MatrixReduceTests.swift

```diff
@@ -0,0 +1,28 @@
+import XCTest
+@testable import Matrix
+
+final class MatrixReduceTests: XCTestCase {
+    func testReduceInto() {
+        let matrix = Matrix(rows: 2, columns: 2, data: [1, 2, 3, 4])
```

Comment:
Sweep: It's wrong initializer. Try to use this initializer: public init(rows: Int, columns: Int, elements: [Element]) throws.
Example:
```
let matrix = try Matrix(rows: 2, columns: 2, elements: [1, 2, 3, 4])
```
And make this test function throws.